docs/build/index.md

```diff
@@ -73,6 +73,12 @@ Ethereum has a large and growing number of tools to help developers build, test,
 
 ## Integrated Development Environments (IDEs)
 
+
+### Visual Studio Code *Professional cross-platform IDE with official Ethereum support.*
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [Azure Blockchain Workbench plugin](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/microsoft-azure-blockchain.azure-blockchain-workbench?tab=Overview)
+- [Code samples](https://github.com/Azure-Samples/blockchain/blob/master/blockchain-workbench/application-and-smart-contract-samples/readme.md)
```

Comment:
This is a powerful third party solidity extension (by a colleague of mine): https://github.com/tintinweb/vscode-solidity-auditor

@virgil, what's your take on adding it as a bullet point here, or would it be best as a stand alone item?

Reply:
Stand alone.  Visual Studio is going to be big.  And it'll make Microsoft happy that they are getting noticed.